pytest/functional/hs_core/test_citations.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import uuid
+
+
+@pytest.mark.django_db
+def test_citation_create(admin_client, resource_for_citation):
+    """
+    Add a unique citation to a resource
+    """
+    citation = str(uuid.uuid4())
+    resource_for_citation.citation = citation
+    assert resource_for_citation.citation == citation
```

Comment:
I don't think this is testing anything?  It's setting an attribute and then testing the attribute was set.  This has nothing to do with the database or any of the code for citations.

Reply:
Indeed. Will write a deeper test. I'll open a new ticket for the edge case problem I ran into, where a database failure was reporting a success in the endpoint, anyway. This realization arose as a part of a total unrelated issue with troubleshooting my environment around full resource deletion and migrations/db permissions.

Reply:
New CRUD tests